ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/regression/LinearRegression.java

```diff
@@ -153,8 +153,8 @@ private void createOptimiser() {
                 optimiser = new RMSProp(learningRate, momentumFactor, epsilon, decayRate);
                 break;
             default:
-                // Use default SGD with a constant learning rate.
-                optimiser = SGD.getSimpleSGD(learningRate, momentumFactor, momentum);
+                // Use AdaGrad by default
+                optimiser = new AdaGrad(learningRate, epsilon);
```

Comment:
Let's add the issue comment link as a comment here for further reference here.

Reply:
Do you mean as a comment in the code with a link?

Reply:
Yes